# Ethereum addresses must be lowercase (no-uppercase-address)

## Rule Details

This rule aims to prevent Ethereum lowercased addresses from causing any harm in a system

<br>

### Examples of **incorrect** code for this rule:

```js
'0XE27BFF97CE92C3E1FF7AA9F86781FDD6D48F5EE9';
```

### Examples of **correct** code for this rule:

```js
'0xe27bff97ce92c3e1ff7aa9f86781fdd6d48f5ee9';
```
